<?xml version="1.0"?>
<launch>
    <node pkg="fake_odometry" type="fake_odometry1" name="fake_odom" output="screen" respawn="false">
        <param name="odom_topic" type="string" value="fake_odom"/>
        <param name="cmd_topic" type="string" value="/cmd_vel_mux/input/navi"/>
        <param name="hz" type="double" value="100"/>
        <param name="initial_x" type="double" value="0.0"/>
        <param name="initial_y" type="double" value="0.0"/>
        <param name="initial_theta" type="double" value="0.0"/>
        <param name="odom_frame" type="string" value="odom"/>
        <param name="child_frame" type="string" value="base_footprint"/>
    </node>
</launch>